src/hooks/useAccounts.tsx

```diff
@@ -188,6 +193,20 @@ const useAccountsContext = () => {
     }
   }, [evmAddress, dydxAddress]);
 
+  const selectedNetwork = useSelector(getSelectedNetwork);
+
+  const chainId = Number(ENVIRONMENT_CONFIG_MAP[selectedNetwork].ethereumChainId);
+
+  const signTypedData = getSignTypedData(selectedNetwork);
+  const { signTypedDataAsync } = useSignTypedData({
```

Comment:
This creates the signing method for generating the EVM signature used for deriving the dYdX wallet. It is also used in [GenerateKeys.tsx](https://github.com/dydxprotocol/v4-web/blob/09aa890ba7640780de2dbc24b09cda8e0c31320b/src/views/dialogs/OnboardingDialog/GenerateKeys.tsx). It should probably abstracted out, to not duplicate the logic. I got lazy during the hackathon :P